include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/err.h>
+#include <linux/netlink.h>
+#include <linux/rwlock.h>
+#include <linux/spinlock.h>
+
+/* Ethernet Principal. */
+#define XIDTYPE_ETHER (__cpu_to_be32(0x22))
+
+#ifdef __KERNEL__
+
+/* ETHER's virtal XID type. */
+int ether_vxt __read_mostly = -1;
+
+/* ETHER context. */
+struct xip_ether_ctx
+{
+	struct net          *net;
+	struct xip_ppal_ctx ctx;
+};
+
+static inline struct xip_ether_ctx *ctx_ether(struct xip_ppal_ctx *ctx)
+{
+	return likely(ctx)
+		? container_of(ctx, struct xip_ether_ctx, ctx)
+		: NULL;
+}
+
+extern int ether_vxt;
+
+#endif /* __KERNEL__ */
+#endif /* _NET_XIA_ETHER_H */
```

Comment:
To be clear, other files have the same issue: net/xia/ppal_ether/Kconfig, 
net/xia/ppal_ether/main.c.

Please, fix this issue, squash the new patches with the proper patch (in this case the first), and rebase. This process simplifies the review process a lot.

I intend to do a careful review tomorrow.